sql/step_04_all_following_steps.sql

```diff
@@ -83,4 +83,4 @@ source step_86_permissions_and_grouppermission.sql;
 source step_87_alter_table_opd.sql;
 source step_88_alter_table_patient.sql;
 source step_89_alter_tables_innodb.sql;
-
+source step_90_login_retries.sql;
```

Comment:
this step should be included also in Dockerfile-ohdb

Reply:
Done